packages/api/src/routers/svc/speech.ts

```diff
@@ -0,0 +1,97 @@
+import express from 'express'
+import cors from 'cors'
+import { corsConfig } from '../../utils/corsConfig'
+import { getRepository } from '../../entity/utils'
+import { getPageById } from '../../elastic/pages'
+import { synthesizeTextToSpeech } from '../../utils/textToSpeech'
+import { Speech, SpeechState } from '../../entity/speech'
+import { UserPersonalization } from '../../entity/user_personalization'
+import { buildLogger } from '../../utils/logger'
+
+const logger = buildLogger('app.dispatch')
+
+export function speechServiceRouter() {
+  const router = express.Router()
+
+  router.options('/', cors<express.Request>({ ...corsConfig, maxAge: 600 }))
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  router.post('/', async (req, res) => {
```

Comment:
I think we need to authenticate this API? I don't see a token validation check here